tests/test_cli.py

```diff
@@ -0,0 +1,145 @@
+"""Unit tests for cli"""
+import json
+import sys
+import unittest
+import unittest.mock as mock
+from datetime import datetime
+
+import geospaas_processing.cli.download as cli_download
+import geospaas_processing.downloaders as downloaders
+from dateutil.tz import tzutc
+from django.contrib.gis.geos import GEOSGeometry
+from freezegun import freeze_time
```

Comment:
Unused imports and imports order.

```suggestion
import sys
import unittest
import unittest.mock as mock
from datetime import datetime

from dateutil.tz import tzutc
from django.contrib.gis.geos import GEOSGeometry
from freezegun import freeze_time

import geospaas_processing.cli.download as cli_download
```

Reply:
Done in the next commit